pkg/config/config_test.go

```diff
@@ -69,7 +69,7 @@ func TestReplicaConfigOutDated(t *testing.T) {
 
 func TestServerConfigMarshal(t *testing.T) {
 	t.Parallel()
-	rawConfig := `{"addr":"192.155.22.33:8887","advertise-addr":"","log-file":"","log-level":"info","log":{"file":{"max-size":300,"max-days":0,"max-backups":0}},"data-dir":"","gc-ttl":86400,"tz":"System","capture-session-ttl":10,"owner-flush-interval":200000000,"processor-flush-interval":100000000,"sorter":{"num-concurrent-worker":4,"chunk-size-limit":999,"max-memory-percentage":30,"max-memory-consumption":17179869184,"num-workerpool-goroutine":16,"sort-dir":"/tmp/sorter"},"security":{"ca-path":"","cert-path":"","key-path":"","cert-allowed-cn":null},"per-table-memory-quota":10485760,"kv-client":{"worker-concurrent":8,"worker-pool-size":0,"region-scan-limit":40}}`
+	rawConfig := `{"addr":"192.155.22.33:8887","advertise-addr":"","log-file":"","log-level":"info","log":{"file":{"max-size":300,"max-days":0,"max-backups":0}},"data-dir":"","gc-ttl":86400,"tz":"System","capture-session-ttl":10,"owner-flush-interval":200000000,"processor-flush-interval":100000000,"sorter":{"num-concurrent-worker":4,"chunk-size-limit":999,"max-memory-percentage":30,"max-memory-consumption":17179869184,"num-workerpool-goroutine":16,"sort-dir":"/tmp/sorter","enable-leveldb-sorter":true,"leveldb-count":16,"leveldb-concurrency":256,"max-open-files":10000,"block-size":65536,"block-cache-size":0,"writer-buffer-size":8388608,"compression":"snappy","target-file-size-base":8388608,"compaction-l0-trigger":160,"write-l0-slowdown-trigger":2147483647,"write-l0-pause-trigger":2147483647,"cleanup-speed-limit":10000},"security":{"ca-path":"","cert-path":"","key-path":"","cert-allowed-cn":null},"per-table-memory-quota":10485760,"kv-client":{"worker-concurrent":8,"worker-pool-size":0,"region-scan-limit":40}}`
```

Comment:
Maybe we should write it to a file, nice long string. 😆

Reply:
It is especially awful when cherry-picking